core/definitions/src/protocols/ibc.ts

```diff
@@ -153,8 +135,8 @@ export function makeGatewayTransferMsg<CN extends Chain>(
   const address =
     typeof recipient === "string"
       ? recipient
-      // @ts-ignore
-      : encoding.b64.encode(recipient.toString());
+      : // @ts-ignore
+        encoding.b64.encode(recipient.toString());
```

Comment:
This one is still whack, `recipient` is `never` type 🤔

Reply:
This makes sense in the context of the definition layer, because at this point, no native address has been registered yet.

Using declaration merging with a global namespace and hence creating a dynamic type of sorts is not something that typescript "really thought of", I believe. It's quite the abuse of the system akin to early stage C++ template metaprogramming, when nobody had yet realized that they were Turing complete until the first guy with a "I bet I can make this baby run doom" mindset came around.